### ISSUE TITLE
Add an Anime page.

### ISSUE THREAD

**Issue by SHARMA-1729** [Fenrir-04/Algo-Media#14]
I would like to add an Anime page which will have curated anime playlist to watch and enjoy.

**Comment by Fenrir-04** [comment]
You mean like adding an anime section in the sidebar ?

**Comment by SHARMA-1729** [comment]
Yes an anime section on sidebar which will have curated playlist frim youtube channels like Muse Asia

**Comment by Fenrir-04** [comment]
sounds good.
go ahead and start working on it.

**Comment by SHARMA-1729** [comment]
I made anime section on sidebar as i said.
